firefox_privacy_notice/sr.md

```diff
@@ -1,71 +1,71 @@
 # Obaveštenje o privatnosti pretraživača Firefox
 
-15\. april 2014.
-{: datetime="2014-04-15" }
+15. april 2014.
```

Comment:
This needs a `\` after the `15` otherwise Markdown will convert it to a list and think it should be "1. april 2014.".